wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -27,6 +28,7 @@
  *
  * <p>This class is provided by the NewCommands VendorDep
  */
+@NoDiscard("Commands must be used! Did you mean to bind it to a trigger?")
```

Comment:
Open to suggestions on the messaging here

Reply:
Oh okay. So perhaps a message like "command instantiated/created but not used" is better?